01_README.md

```diff
@@ -0,0 +1,31 @@
+![Singer Logo](https://trello-attachments.s3.amazonaws.com/58c8696247956895aea87ef2/58d2d15c8baaf0c33f36c87f/a789e41241329c5b972a6e105e954543/Screen_Shot_2017-04-27_at_3.58.09_PM.png)
```

Comment:
Changing the name from `README.md` may break Github's ability to display it on the main page of the repo.

When I visit https://github.com/ashhath/getting-started, there's no README displayed there.